index.html

```diff
@@ -1291,8 +1291,22 @@
             </li>
             <li>Return <em>P</em>.
             </li>
+            <li>If the user agent knows there will be no available presentation
```

Comment:
If the user agent is unable to monitor presentation displays for the entire duration of the controlling browsing context (e.g., because the user has disabled this feature), then: